src/book.py

```diff
@@ -82,7 +82,37 @@ def __init__(self, filepath=None, rawtext=None, encoding='utf-8', file_format='s
                 else:
                     self._title = None
         else:
-            self._title = None
+            self._title = title #FIXME IN MASTER
+            
+    def __add__(self, other): 
```

Comment:
Is there a function inside BookText that uses, or will use, this function? Methods that start with underscores are by convention only meant for internal use.

Reply:
This is also the only way I know how to overload the '+' operator. For reference, https://www.geeksforgeeks.org/operator-overloading-in-python/

Reply:
Oh I see. I was misunderstanding what it was doing. Very clever.